### PR TITLE
Move test fixtures to test/fixtures making it obvious what each direc…

### PR DESCRIPTION
**Pull request by JoeChapman**
…tory is.

General tidy up because test/ was becoming a mess.

### CODE REVIEW

File: test/integration/index.js [hunk 1]
```diff
@@ -75,7 +76,7 @@ describe('bootstrap()', () => {
       routes: [{
         steps: {},
       }]
-    })).should.Throw('Cannot find fields at ' + path.resolve(__dirname, '../../test/not_a_valid_path'))
+    })).should.Throw('Cannot find fields at ' + root + '/not_a_valid_path')
```
Comment (joefitter) [inline test/integration/index.js]:
```js
`Cannot find fields at ${root}/not_a_valid_path`
```
would be nicer

File: test/integration/index.js [hunk 2]
```diff
@@ -84,7 +85,7 @@ describe('bootstrap()', () => {
         steps: {},
         fields: 'not_a_valid_path'
       }]
-    })).should.Throw('Cannot find route fields at ' + path.resolve(__dirname, '../../test/not_a_valid_path'))
+    })).should.Throw('Cannot find route fields at ' + root + '/not_a_valid_path')
```
Comment (joefitter) [inline test/integration/index.js]: :nauseated_face:

Comment (JoeChapman) [inline test/integration/index.js]: fair point

**Comment by JoeChapman** [comment]
@joefitter it's alright

**Comment by JoeChapman** [comment]
@joefitter happy now?

**Pull request merged**